### DISCUSSION TITLE
CUDA vs OpenCL

### DISCUSSION THREAD

**Discussion by hajes** [madMAx43v3r/chia-gigahorse#118] [General]
Could you explain why not use OpenCL that is platform-independent, please?

I guess CUDA is more advanced for programming, but bound to nVidia only.

**Comment by AtomicInternet2** [comment]
chia-gigahorse is the only Chia harvester that supports OpenCL.  

nVidia CUDA simply has better programming support / easier to develop for.

The answer is better support/ecosystem for CUDA.
